examples/ChatQnA/deploy/gaudi.md

```diff
@@ -0,0 +1,974 @@
+|UI                   |              | NA                       | Gateway Service |
+
+Tools and models mentioned in the table are configurable either through the
+environment variable or `compose.yaml` file.
+:::
+::::
+
+Set the necessary environment variables to setup the use case case
+
+> Note: Replace `host_ip` with your external IP address. Do **NOT** use localhost
+> for the below set of environment variables
+
+### Dataprep
+
+    export DATAPREP_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/dataprep"
+    export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_file"
+    export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_file"
+
+  ### VectorDB
+
+    export REDIS_URL="redis://${host_ip}:6379"
+    export INDEX_NAME="rag-redis"
+
+  ### Embedding Service
+
+    export EMBEDDING_MODEL_ID="BAAI/bge-base-en-v1.5"
+    export EMBEDDING_SERVICE_HOST_IP=${host_ip}
+    export RETRIEVER_SERVICE_HOST_IP=${host_ip}
+    export TEI_EMBEDDING_ENDPOINT="http://${host_ip}:8090"
+
+### Reranking Service
+
+    export RERANK_MODEL_ID="BAAI/bge-reranker-base"
+    export TEI_RERANKING_ENDPOINT="http://${host_ip}:8808"
+    export RERANK_SERVICE_HOST_IP=${host_ip}
+
+### LLM Service
+
+::::{tab-set}
+:::{tab-item} vllm
+:sync: vllm
+
+    export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+    export LLM_SERVICE_HOST_IP=${host_ip}
+    export LLM_SERVICE_PORT=9000
+    export vLLM_LLM_ENDPOINT="http://${host_ip}:8007"
+:::
+:::{tab-item} TGI
+:sync: TGI
+
+    export LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+    export LLM_SERVICE_HOST_IP=${host_ip}
+    export LLM_SERVICE_PORT=9000
+    export TGI_LLM_ENDPOINT="http://${host_ip}:8005"
+:::
+::::
+
+### Megaservice
+
+    export MEGA_SERVICE_HOST_IP=${host_ip}
+    export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/chatqna"
+
+### Gaurdrails(optional)
+If guardrails microservice is enabled in the pipeline, the below environment variables are necessary to be set.
+```
+export GURADRAILS_MODEL_ID="meta-llama/Meta-Llama-Guard-2-8B"
+export SAFETY_GUARD_MODEL_ID="meta-llama/Meta-Llama-Guard-2-8B"
+export SAFETY_GUARD_ENDPOINT="http://${host_ip}:8088"
+export GUARDRAIL_SERVICE_HOST_IP=${host_ip}
+```
+## Deploy the use case
+
+In this tutorial, we will be deploying via docker compose with the provided
+YAML file.  The docker compose instructions should be starting all the
+above mentioned services as containers.
+
+::::{tab-set}
+:::{tab-item} vllm
+:sync: vllm
+
+```
+cd GenAIExamples/ChatQnA/docker_compose/intel/hpu/gaudi
+docker compose -f compose_vllm.yaml up -d
+```
+:::
+:::{tab-item} TGI
+:sync: TGI
+
+```
+cd GenAIExamples/ChatQnA/docker_compose/intel/hpu/gaudi
+```
+
+Follow ONE of the methods below.
+1. Use TGI for the LLM backend.
+
+```bash
+docker compose -f compose.yaml up -d
+```
+
+2. Enable the Guardrails microservice in the pipeline. It will use a TGI Guardrails service.
+
+```bash

```

Comment:
Same here. Indent these three lines to keep the code block within the ordered list.

Reply:
done, the code blocks appear formatted a code in the readme